src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1458,11 +1458,25 @@ private Resource makeValidHypothesis(Resource... resources) {
         // Exactly 1 hypothesis should exist in model
         @Nested
         class SingleHypothesis {
+            Resource event;
```

Comment:
event, eventEdge, and all their setup will be required for each hypothesis test class. This should all be moved up into NISTHypothesisExamples and NISTHypothesisExamples.setup().

Reply:
I think doing this will have conflicts with HypothesisEdgeImportanceValue() and HypothesisRequiredOneEventOrRelationWithOneEdge() tests. For those two sets of test you want to have control over when events/relations and their respective edges are created. That is why I ended up putting this within each test class. Let me know if you feel differently or if you want to talk more about it.